daemon/api.go

```diff
@@ -973,6 +993,10 @@ func snapsOp(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("cannot decode request body into snap instruction: %v", err)
 	}
 
+	if inst.hasOpts() {
```

Comment:
Also, probably `"unsupported option provided for multi-snap operation"` for the same reason.